aruco_markers:
  - id: 0
    parent_link: "R5A_link1"
    translation: [0, 0, 0.1]
    rotation: [0, 0, 0]  # Roll, Pitch, Yaw in radians
  - id: 1
    parent_link: "R5A_link1"
    translation: [0, 0, -0.1]
    rotation: [0, 0, 0]
  - id: 2
    parent_link: "R5A_link1"
    translation: [0.1, 0, 0]
    rotation: [0, 1.5708, 0]
  - id: 3
    parent_link: "R5A_link1"
    translation: [-0.1, 0, 0]
    rotation: [0, -1.5708, 0]
  - id: 4
    parent_link: "R5A_link2"
    translation: [0, 0, 0.1]
    rotation: [0, 0, 0]
  - id: 5
    parent_link: "R5A_link2"
    translation: [0, 0, -0.1]
    rotation: [0, 0, 0]
  - id: 6
    parent_link: "R5A_link2"
    translation: [0.1, 0, 0]
    rotation: [0, 1.5708, 0]
  - id: 7
    parent_link: "R5A_link2"
    translation: [-0.1, 0, 0]
    rotation: [0, -1.5708, 0]
  - id: 8
    parent_link: "R5A_link3"
    translation: [0, 0, 0.05]
    rotation: [0, 0, 0]
  - id: 9
    parent_link: "R5A_link3"
    translation: [0, 0, -0.05]
    rotation: [0, 0, 0]
  - id: 10
    parent_link: "R5A_link3"
    translation: [0.1, 0, 0]
    rotation: [0, 1.5708, 0]
  - id: 11
    parent_link: "R5A_link3"
    translation: [-0.1, 0, 0]
    rotation: [0, -1.5708, 0]
  - id: 12
    parent_link: "R5A_link4"
    translation: [0, 0, 0.05]
    rotation: [0, 0, 0]
  - id: 13
    parent_link: "R5A_link4"
    translation: [0, 0, -0.05]
    rotation: [0, 0, 0]
  - id: 14
    parent_link: "R5A_link4"
    translation: [0.05, 0, 0]
    rotation: [0, 1.5708, 0]
  - id: 15
    parent_link: "R5A_link4"
    translation: [-0.05, 0, 0]
    rotation: [0, -1.5708, 0]
